frontend/src/bundles/common/components/icon/enums/icon-components.enum.ts

```diff
@@ -1,6 +1,12 @@
 import ArrowDownIcon from '~/assets/img/icons/arrow-down-icon.svg?react';
+import ScheduleIcon from '~/assets/img/icons/calendar-days-icon.svg?react';
+import GoalsIcon from '~/assets/img/icons/goals-icon.svg?react';
+import HelpIcon from '~/assets/img/icons/help-icon.svg?react';
 import LogoIcon from '~/assets/img/icons/logo-icon.svg?react';
+import LogoutIcon from '~/assets/img/icons/logout-icon.svg?react';
 import NotFoundIcon from '~/assets/img/icons/not-found-icon.svg?react';
+import OverviewIcon from '~/assets/img/icons/overview-icon.svg?react';
+import WorkoutIcon from '~/assets/img/icons/workout-icon.svg?react';
```

Comment:
Since we already have Heroicons in the project, do we really need all these SVGs? It seems like common icons that should be available in the library.

Reply:
Yes, I agree with your point.

The thing is, as you could see, I had a task - create only Base layout.

And this remark applies more to the Sidebar component (which I simply merged into my branch in order to implement the base layout).

Since this PR ([Sidebar](https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-lime/pull/52)) is not approved at this stage, it is not advisable for me to change anything in the Sidebar yet.

But is it possible to tell the author of this PR (Sidebar) about your comment?

Reply:
@VasylynaKolodiy Hi! Alright i will change the icons available to heroicons and the rest like on svgrs!